app/src/main/java/com/example/habitsmasher/FollowUserDialog.java

```diff
@@ -91,9 +99,17 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                                         Log.d(TAG, "Username exists");
 
                                         // if username exists, update follower/following counts
-                                        String currentUserId = getCurrentUserId();
+                                        String currentUserId = UserDatabaseHelper.getCurrentUser(_context).getId();
+                                        String currentUserUsername = UserDatabaseHelper.getCurrentUser(_context).getUsername();
```

Comment:
I extracted the `getCurrentUserId()` method to the `UserDatabaseHelper`. So we can make a call to that class instead!